### DISCUSSION TITLE
How to begin?

### DISCUSSION THREAD

**Discussion by ericfreitas88** [hyochan/react-native-iap#2741] [Q&A]
Hey, guys. How are you?

I need help implementing this library in my project.

I tried to read the documentation for this library, but I can't understand it.

I would like to understand this step by step.

I searched the internet, but didn't find any current step-by-step content.

I don't speak English very well, but I would be very happy if someone could help me.

I've been working with React Native for a little over a month, so...
Many things are still a mystery to me.

**Comment by hyochan** [comment]
Have you searched through the youtube?
https://www.youtube.com/results?search_query=react-native-iap

**Reply by ericfreitas88** [reply]
Hello  @hyochan. Yes, I searched for several days.

In fact, I already watched the first videos on the list you sent me, but I couldn't understand them.

I try to read the documentation but I don't understand anything.

I noticed that there isn't much content on the internet about this, especially in Portuguese.

Can you help me understand how this library works?
